templates/demo/app/dapp.js

```diff
@@ -1,161 +1,75 @@
-/*globals $, SimpleStorage, document*/
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Tabs, Tab } from 'react-bootstrap';
 
-import $ from 'jquery';
-import 'bootstrap';
-import 'bootstrap/dist/css/bootstrap.min.css';
 import EmbarkJS from 'Embark/EmbarkJS';
-import SimpleStorage from 'Embark/contracts/SimpleStorage';
+import Blockchain from './components/blockchain';
+import Whisper from './components/whisper';
+import Storage from './components/storage';
 
 import './dapp.css';
 
-var addToLog = function(id, txt) {
-  $(id + " .logs").append("<br>" + txt);
-};
+class App extends React.Component {
 
-// ===========================
-// Blockchain example
-// ===========================
-$(document).ready(function() {
+  constructor(props) {
+    super(props);
 
-  $("#blockchain button.set").click(function() {
-    var value = parseInt($("#blockchain input.text").val(), 10);
-
-    // If web3.js 1.0 is being used
-    if (EmbarkJS.isNewWeb3()) {
-      SimpleStorage.methods.set(value).send({from: web3.eth.defaultAccount});
-      addToLog("#blockchain", "SimpleStorage.methods.set(value).send({from: web3.eth.defaultAccount})");
-    } else {
-      SimpleStorage.set(value);
-      addToLog("#blockchain", "SimpleStorage.set(" + value + ")");
+    this.state = {
+      whisperEnabled: false,
+      storageEnabled: false
     }
```

Comment:
In https://github.com/embark-framework/embark/pull/390 (not yet approved/merged), this can be achieved with `EmbarkJS.Storage.isAvailable()` and `EmbarkJS.Messages.isAvailable()`